src/threadpool-utils.h

```diff
@@ -112,11 +112,6 @@ static inline size_t divide_round_up(size_t dividend, size_t divisor) {
 	}
 }
 
-/* Windows headers define min and max macros; undefine it here */
-#ifdef min
-	#undef min
-#endif
```

Comment:
This is still needed. I've seen an SDK that completely disregards NOMINMAX when building in C mode.

Reply:
Yes I found that in testing as well.
It doesn't seem to be an SDK issue.
I checked my Windows SDK respects `NOMINMAX`, but stdlib.h seems to define another `min`.